Makefile

```diff
@@ -13,7 +11,8 @@ PREFIX                       := usr/local
 INSTALL_BIN                  := $(DESTDIR)/$(PREFIX)/bin
 INIT_PREFIX                  := $(DESTDIR)/etc/init.d
 SYSTEMD_PREFIX               := $(DESTDIR)/lib/systemd/system
-CONFIG_PREFIX                := $(DESTDIR)/etc/keyless
+CONFIG_PATH                  := etc/keyless
```

Comment:
Shouldn't CONFIG_PATH also be relative to DESTDIR?

Reply:
Then I don't understand why config_path is used instead of config_prefix in lines 65 and 77.

Reply:
B/c those commands are referring to the file path after installation. The context is different.

Reply:
Wait, isn't the purpose of DESTDIR to, for instance, allow the user to install under /usr/local/etc/keyless? If not then never mind, I misunderstood this.

Reply:
DESTDIR is set directly above here